go.mod

```diff
@@ -3,36 +3,25 @@ module github.com/kubernetes-csi/volume-data-source-validator
 go 1.15
 
 require (
-	github.com/golang/protobuf v1.4.3 // indirect
-	github.com/google/go-cmp v0.5.2 // indirect
-	github.com/google/uuid v1.1.2 // indirect
 	github.com/googleapis/gnostic v0.5.1 // indirect
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/imdario/mergo v0.3.10 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.9.1
-	github.com/kubernetes-csi/external-snapshotter/client/v3 v3.0.0
+	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.0.0
 	github.com/kubernetes-csi/volume-data-source-validator/client v0.0.0-00010101000000-000000000000
 	github.com/onsi/ginkgo v1.14.1 // indirect
 	github.com/onsi/gomega v1.10.2 // indirect
-	golang.org/x/crypto v0.0.0-20201208171446-5f87f3452ae9 // indirect
-	golang.org/x/net v0.0.0-20201209123823-ac852fbbde11 // indirect
 	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5 // indirect
-	golang.org/x/sys v0.0.0-20201214095126-aec9a390925b // indirect
-	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
-	golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	gopkg.in/yaml.v2 v2.4.0 // indirect
-	k8s.io/api v0.19.9
-	k8s.io/apimachinery v0.19.9
-	k8s.io/client-go v0.19.9
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
 	k8s.io/klog/v2 v2.8.0
-	k8s.io/utils v0.0.0-20200912215256-4140de9c8800 // indirect
 )
 
 replace (
 	github.com/kubernetes-csi/volume-data-source-validator/client => ./client
-	k8s.io/api => k8s.io/api v0.19.9
-	k8s.io/apimachinery => k8s.io/apimachinery v0.19.9
-	k8s.io/client-go => k8s.io/client-go v0.19.9
-	k8s.io/code-generator => k8s.io/code-generator v0.19.9
+	k8s.io/api => k8s.io/api v0.21.0
+	k8s.io/apimachinery => k8s.io/apimachinery v0.21.0
+	k8s.io/client-go => k8s.io/client-go v0.21.0
```

Comment:
Can you see if these k8s.io replacements can be removed?  I think they can be removed if go.mod does not need k8s.io/kubernetes.

Reply:
You're right.